src/types/types.tsx

```diff
@@ -94,7 +111,14 @@ export interface ItokenCommunity {
   type: EcommunityType;
   name: string;
   imageUrl: string | undefined;
-  communityData: IcommunityData | undefined;
+  categories: Icategory[] | undefined;
+}
+
+export interface ICustomCommunity extends ItokenCommunity {
+  displayName: string;
+  tokens: Array<string>; // for query
+  requirements: string[];
+  managers: Array<string>;
```

Comment:
Let's use same syntac, either string[] or Array<string>

Reply:
???

Reply:
Like this:
tokens: string[];
requirements: string[];
managers: string[];

OR:

tokens: Array<string>;
requirements: Array<string>;
managers: Array<string>;

Reply:
old commit